ezbake/config/conf.d/auth.conf

```diff
@@ -137,6 +137,15 @@ authorization: {
             name: "puppetlabs static file content"
         },
         {
+            match-request: {
+                path: "/puppet/v3/tasks"
+                type: path
```

Comment:
Yeah, I don't expect we'll have anything that needs further restriction with regards to this endpoint, so even though it doesn't work, a 404 seems like a more clear result than saying you're not authorized to access something that turns out to be nothing